lib/deck.rb

```diff
@@ -0,0 +1,17 @@
+class Deck 
+  attr_reader :cards
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def count
+    count = cards.count
+  end
+  
+  def cards_in_category(name)
+    groups = cards.group_by do |card| 
+      card.category 
+    end
+    groups[name]
```

Comment:
Nice use of `group_by`. I like this implementation! Other enumerable methods that work well here are `select` and `find_all`